lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -520,15 +520,7 @@ def test_get_html_with_existing_edx_video_id(self):
         # create test profiles and their encodings
         encoded_videos = []
         for profile, extension in [("desktop_webm", "webm"), ("desktop_mp4", "mp4")]:
```

Comment:
These happen to match real values, but the test can put in whatever arbitrary strings it wants for these.